suite-native/home-graph/src/components/TimeSwitchItem.tsx

```diff
@@ -32,7 +32,7 @@ export const TimeSwitchItem = ({ value, shortcut }: TimeSwitchItemProps) => {
     };
 
     return (
-        <TouchableOpacity onPress={() => handleSelectTimeFrame(value)}>
+        <TouchableOpacity onPress={() => handleSelectTimeFrame(value)} testID={value}>
```

Comment:
use something like `TimeSwichItem_${value}` for testID

Reply:
https://github.com/trezor/trezor-suite/pull/5862/commits/9901647f9369f5be6c73234fa3306f562bc425b9